.github/workflows/cd.yml

```diff
@@ -1,13 +1,16 @@
 name: cd
 on:
   workflow_run:
-    workflows: ["ci"]
-    branches-ignore: ["*"]
+    workflows:
+      - ci
+    branches:
+      - master
+      - main
```

Comment:
Does that mean the `cd` workflow only runs on these branches? Or does it also run on tags?

Reply:
It will run on tags also since it has the directive below
```yaml
  push:
    tags:
      - "v[0-9]+.[0-9]+.[0-9]+"
```